avg_age.js

```diff
@@ -0,0 +1,33 @@
+var mongoose = require("mongoose");
+var passwordHash = require("password-hash");
+var {conn, usersprofile_schema} = require('./index');
+
+var avg_age = conn.model("usersProfile", usersprofile_schema);
```

Comment:
already defined this model

Reply:
removed this model and used existing  one